scripts/run_all_tests.sh

```diff
@@ -46,29 +46,42 @@ then
     echo "Successfully Downloaded all necessary vector test files"
 fi
 
+# Check if pgvector is available
+pgvector_installed=$($PSQL -U $DB_USER -c "SELECT 1 FROM pg_available_extensions WHERE name = 'vector'" -tA)
+
+# Generate schedule.txt
+rm -rf $TMP_OUTDIR/schedule.txt
+if [ -n "$FILTER" ]; then
```

Comment:
just curious, what is the difference between old `-z` and new `-n` ?

Reply:
-z and -n are pretty much "is null" vs "is not null". I was originally planning to do "if filter is not null then X else if pgvector then Y else Z". I didn't take that route, but this was just a side effect of that.